src/Mouf/Mvc/Splash/Services/SplashUtils.php

```diff
@@ -201,9 +201,31 @@ public static function buildControllerResponse($callback)
         }
 
         $code = http_response_code();
-        $headers = headers_list();
+        $headers = SplashUtils::greatResponseHeaders();
+
+        // Suppress actual headers (re-add by Symfony Response)
+        // If you don't remove old headers, it's duplicated in HTTP Headers
+        foreach ($headers as $key => $head) {
+            header_remove($key);
+        }
```

Comment:
Thx for you comment. 

I don't use header_remove without parameters because php.net indicate : 
```
Caution
This function will remove all headers set by PHP, including cookies, session and the X-Powered-By headers.
```